open-metadata/OpenMetadata
# ISSUE-11035: add ZH ElasticSearch index mapping

### Describe your changes:

Fixes 11035


changes:
- add zh index mapping json files under `openmetadata-service/src/main/resources/elasticsearch`
- add ZH enum value to `openmetadata-spec/src/main/resources/json/schema/configuration/elasticSearchConfiguration.json`

test:
- `mvn clean install --DskipTests` successfully
-  change env `ELASTICSEARCH_INDEX_MAPPING_LANG` to `${ELASTICSEARCH_INDEX_MAPPING_LANG:-ZH}` in docker/development/docker-compose.yml. change docker image of elasticseach to `elasticsearch-ik:7.11.2` with ik analyzer pre-integrated
- docker compose -f docker/development/docker-compose.yml up --build -d
- Re index all  with ZH language on page http://localhost:8585/settings/openMetadata/search/on_demand
- search chinese words. and search results is as expected.


#
### Type of change:


#
### Checklist:



- [x] For connector/ingestion changes: I updated the documentation.
